py2many/cli.py

```diff
@@ -90,6 +90,15 @@ def go_settings(args):
     return LanguageSettings(GoTranspiler(), ".go", "gofmt -w")
 
 
+def _process_once(settings, filename, outdir):
+    output_path = outdir / (filename.stem + settings.ext)
+    print(f"{filename}...{output_path}")
+    with open(output_path, "w") as f:
+        source_data = open(filename).read()
```

Comment:
File handle leak here (leak present in main branch as well). Using a context manager should fix.